src/modeler/AutoRest.Swagger/Validation/ProvidersPathValidation.cs

```diff
@@ -13,10 +13,10 @@
 
 namespace AutoRest.Swagger.Validation 
 {
-    public class ProviderPathsValidation : TypedRule<Dictionary<string, Dictionary<string, Operation>>>
+    public class ProvidersPathValidation : TypedRule<Dictionary<string, Dictionary<string, Operation>>>
     {
-        private readonly Regex FullRegex = new Regex(@"\/Subscriptions\/\{.+\}\/ResourceGroups\/\{.+\}\/providers\/[^\/]+(\/[^\/]+\/\{[^\}]+\})*(\/[\/^])?([\?]?[^\/]+)*$", RegexOptions.IgnoreCase);
-        private readonly Regex ProviderRegex = new Regex(@"Subscriptions\/\{.+\}\/ResourceGroups\/\{.+\}\/providers\/.+$", RegexOptions.IgnoreCase);
+        private Regex FullRegex = new Regex(@"\/Subscriptions\/\{.+\}\/ResourceGroups\/\{.+\}\/providers\/[^\/]+(\/[^\/]+\/\{[^\}]+\})*(\/[^\/]+)?([\?]?[^\/]+)*$", RegexOptions.IgnoreCase);
```

Comment:
regex should be equivalent to `\/Subscriptions\/\{.+\}\/ResourceGroups\/\{.+\}\/providers\/[^\/]+(\/[^\/]+\/\{[^\}]+\})*(\/[^\/]+)?$` (shorter)